docker/checkout.bash

```diff
@@ -38,12 +38,12 @@ done
 cd ${dir}/src
 
 if [[ "$BRANCH" = "develop" ]]; then
-      git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-msgs.git --branch $BRANCH
+      git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-msgs.git --branch integration/routing
```

Comment:
This change was correct for integration testing, but I don't want it commited since it breaks the expected behavior of the -d option please change it back to $BRANCH